cmd/mrregister.cpp

```diff
@@ -445,8 +445,11 @@ void run () {
       case 2:
         rigid_estimator = Registration::LP;
         break;
-      default:
```

Comment:
Will have to back-trace where this difference has arisen from and eradicate it from this PR.

Reply:
Extracted and proposed in #2700; should merge that to `dev` before PR draft status is removed.